source/app/pages/select.js

```diff
@@ -68,16 +68,16 @@ else {
 
       <h2>Upload your own documents</h2>
       <FileUpload />
-      <h2>Or add some example documents</h2>
-      <SampleCollections />
+      {/* <h2>Or add some example documents</h2>
```

Comment:
Is there a reason to remove the Sample documents? I believe we would still need this to be present for showing the other (existing) capabilities of DUS